lib/pengine/rules.c

```diff
@@ -1000,15 +1000,25 @@ compare_attr_expr_vals(const char *l_val, const char *r_val, const char *type,
             cmp = strcasecmp(l_val, r_val);
 
         } else if (pcmk__str_eq(type, "number", pcmk__str_casei)) {
-            int l_val_num = crm_parse_int(l_val, NULL);
-            int r_val_num = crm_parse_int(r_val, NULL);
+            long long l_val_num = crm_parse_ll(l_val, NULL);
+            int rc1 = errno;
+
+            long long r_val_num = crm_parse_ll(r_val, NULL);
+            int rc2 = errno;
```

Comment:
I'd say the crm_parse_ll() makes this a feature (for the commit message, i.e. expanding the range of node attribute comparisons).

Reply:
Fixed